css/tabbed-content.styl

```diff
@@ -68,11 +68,10 @@
   &.social-icon
     padding: 0.5em
 
+.tabbed-content .subject-page-tabs
   button.link-style
-    &.create-discussion-or-comment
-      text-decoration: none
-      color: COPY_GREY_DARK
-      text-transform: uppercase
-
-      &:focus
-        outline: none
+    text-decoration: none
```

Comment:
Just a tiny comment on convention: Could the CSS properties be put in alphabetic order?

Reply:
Sure, no problem :)